Assignments/Dockerfile

```diff
@@ -0,0 +1,20 @@
+
+FROM mariadb:latest
+
+ENV PACKAGES openssh-server openssh-client
+RUN apt-get update && apt-get install -y $PACKAGES
+
+COPY baseball.sql .
```

Comment:
Don't you need destinations here? I'm not really sure, I was kind of going off of stack overflow for these.